docs/site/content/docs/latest/assets/cli-install-mac.md

```diff
@@ -26,7 +26,7 @@ You can install Tanzu Community Edition using Homebrew **or** you can download t
 
     ```sh
     cd <DOWNLOAD-DIR>
-    tar xzvf tce-darwin-amd64-{{< release_latest >}}.tar.gz
+    tar xzvf ~/tce-darwin-amd64-{{< release_latest >}}.tar.gz
```

Comment:
The `~/` assumes you extracted everything in your homedir. I think the correct think would be extract in the directory you downloaded into.